src/misc/lv_timer_private.h

```diff
@@ -33,7 +33,7 @@ struct _lv_timer_t {
     lv_timer_cb_t timer_cb;    /**< Timer function */
     void * user_data;          /**< Custom user data */
     int32_t repeat_count;      /**< 1: One time;  -1 : infinity;  n>0: residual times */
-    uint32_t paused : 1;
+    volatile uint8_t paused;
```

Comment:
```suggestion
    volatile int paused;
```

Reading/Writing `uint8_t` might not be atomic.